source/linux/Foundational_Components/Kernel/Kernel_Drivers/Storage/MMC-SD.rst

```diff
@@ -1,22 +1,73 @@
 .. http://processors.wiki.ti.com/index.php/Linux_Core_MMC/SD_User%27s_Guide
 
-MMC/SD
-######
+MMCSD
+#####
+
+.. ifconfig:: CONFIG_part_family in ('AM62X_family', 'AM62PX_family')
+
+   .. warning::
+
+      There is important information on multimedia card (MMC) support for |__PART_FAMILY_DEVICE_NAMES__| device, go
+      :ref:`here <mmc-support-linux>` for more information.
+
+Acronyms and definitions
+************************
+
++---------------------------------------------------+
+| Acronyms in this guide                            |
++===================================================+
+| Multimedia card secure digital (MMCSD)            |
++---------------------------------------------------+
+| Multimedia card (MMC)                             |
++---------------------------------------------------+
+| Embedded multimedia card (eMMC)                   |
++---------------------------------------------------+
+| Secure digital (SD)                               |
++---------------------------------------------------+
+| Secure digital input/output (SDIO)                |
++---------------------------------------------------+
+| User data area (UDA)                              |
++---------------------------------------------------+
+| Secure digital high capacity (SDHC)               |
++---------------------------------------------------+
+| Ultra high speed (UHS)                            |
++---------------------------------------------------+
+| Local host (LH)                                   |
++---------------------------------------------------+
+| Digital signal processor (DSP)                    |
++---------------------------------------------------+
+| Dual data rate (DDR)                              |
++---------------------------------------------------+
+| Single data rate (SDR)                            |
++---------------------------------------------------+
+| Filesystem (FS)                                   |
++---------------------------------------------------+
+| Root filesystem (rootfs)                          |
++---------------------------------------------------+
+| Unmount (umount)                                  |
++---------------------------------------------------+
+| Fixed disk (fdisk)                                |
++---------------------------------------------------+
+| Virtual file allocation table (vfat)              |
++---------------------------------------------------+
+| Fourth extended file system (ext4)                |
++---------------------------------------------------+
+| Microprocessor unit (MPU)                         |
++---------------------------------------------------+
+| Joint electron device engineering council (JEDEC) |
++---------------------------------------------------+
```

Comment:
I do not want to see other sections adopting this. Vale requires only a definition for the first acronym. In addition, some of these are actually commands. In this case they should be added as case sensitive words to the current dictionary: https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#vale-grammar-spelling-and-prose-checking